tests/test_modeling_common.py

```diff
@@ -2217,9 +2217,12 @@ def check_device_map_is_respected(self, model, device_map):
     @require_accelerate
     @require_torch_gpu
     def test_disk_offload(self):
+        if all([model_class._no_split_modules is None for model_class in self.all_model_classes]):
+            return
```

Comment:
Exit early if no model should be tested to avoid failure later (for Funnel)

Reply:
OK.

(Me too overthinking - feel free to ignore)
- Not sure if it's possible: `no_split_modules` is not None but `num_hidden_layers` couldn't be set ..?

Reply:
Not currently, as the models that implement `_no_split_modules` are OPT, GPT-2, GPT-Neo, GPT-J and T5.